test/controllers/licences.controller.test.js

```diff
@@ -538,7 +538,7 @@ describe('Licences controller', () => {
           const response = await server.inject(options)
 
           expect(response.statusCode).to.equal(200)
-          expect(response.payload).to.contain('You’ve marked this licence for the next supplementary bill run')
+          expect(response.payload).to.contain('You&#39;ve marked this licence for the next supplementary bill run')
```

Comment:
```suggestion
          expect(response.payload).to.contain("You've marked this licence for the next supplementary bill run")
```

Reply:
<img width="865" alt="image" src="https://github.com/user-attachments/assets/ae5bcaef-e4f9-4826-92e5-2f98fafd8da6">
Doing it that way doesn't work because it reads it from the payload and in the payload it shows as 'You&#39;ve'. 
There are other controllers that have had the same issue where they have used 'You&#39;ve' instead of 'You've'

<img width="575" alt="image" src="https://github.com/user-attachments/assets/6e839001-9f04-417f-82fc-f94ee1b3317c">